content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md

```diff
@@ -4,32 +4,34 @@ reviewers:
 title: Upgrading kubeadm clusters
 content_type: task
 weight: 20
-min-kubernetes-server-version: 1.19
+min-kubernetes-server-version: "{{< skew latestVersion >}}"
```

Comment:
Does this actually render OK? To check, add the `{{< version-check >}}` shortcode into the page.

Reply:
how is `min-kubernetes-server-version` used on this page and do we need it?
when i was checking the render yesterday i did not see indications of it.

Reply:
`min-kubernetes-server-version` is used for `{{< version-check >}}` and is also available for use elsewhere.

I think given this PR and its intent, I would remove that line from the YAML.

Reply:
removed.

Reply:
@fabriziopandini as @sftim noted it's used with the `{{< version-check >}}` functionality to indicate to users if a certain feature requires a minimum k8s version.

it's not needed in the doc for the time being.